dask/array/image.py

```diff
@@ -0,0 +1,55 @@
+from .core import Array
+from glob import glob
+import os
+try:
+    from skimage.io import imread as sk_imread
+except ImportError:
+    pass
+
+
+def tokenize(*args):
+    from hashlib import md5
+    return md5(str(args).encode()).hexdigest()
+
+def add_leading_dimension(x):
+    return x[None, ...]
+
+
+def imread(filename):
+    """ Read a stack of images into a dask array
```

Comment:
FYI skimage is investigating transitioning to imageio instead of our
current I/O infrastructure. If you want the cleanest dependency structure,
using imageio directly might be worth looking into (despite how much I love
increased use of skimage).

CC @almarklein
On Aug 13, 2015 11:52 AM, "Matthew Rocklin" <notifications@github.com>
wrote:

> In dask/array/image.py
> <https://github.com/ContinuumIO/dask/pull/578#discussion_r36997134>:
>
> > +try:
> > +    from skimage.io import imread as sk_imread
> > +except ImportError:
> > +    pass
> > +
> > +
> > +def tokenize(*args):
> > +    from hashlib import md5
> > +    return md5(str(args).encode()).hexdigest()
> > +
> > +def add_leading_dimension(x):
> > +    return x[None, ...]
> > +
> > +
> > +def imread(filename):
> > +    """ Read a stack of images into a dask array
>
> Fixed.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ContinuumIO/dask/pull/578/files#r36997134>.
>